tests/test_get_session.py

```diff
@@ -35,15 +38,22 @@ async def load_session(self, request):
         await get_session(req)
 
 
-async def test_get_new_session():
+async def test_get_new_session() -> None:
     req = make_mocked_request('GET', '/')
     session = Session('identity', data=None, new=False)
 
     class Storage(AbstractStorage):
-        async def load_session(self, request):
+        # Ignoring typing since return type is on purpose wrong
+        @no_type_check
```

Comment:
Use a `type: ignore[code]` instead for same reasons above.
We could probably also just change `pass` to `return {}` or similar and avoid the errors altogether.